source/language/classical.rst

```diff
@@ -427,6 +427,278 @@ Terminating the program early
 The statement ``end;`` immediately terminates the program, no matter what scope
 it is called from.
 
+
+The Switch statement
+--------------------
+
```

Comment:
Can we please start the definition of the switch statement with a brief description of its functionality/purpose, ie.,  for `while` the equivalent is

> The statement while ( bool ) <body> executes program until the Boolean evaluates to false [[3]](https://github.com/openqasm/openqasm/blob/2ed64cbf4a5cb8ebe29fae11d8a4f0a82f8ca790/source/language/classical.rst#id11). Variables in the loop condition statement may be modified within the while loop body. The body can be either a single statement terminated by a semicolon, or a program block in curly braces {} of several statements:

Reply:
Done.